setup.py

```diff
@@ -10,14 +10,14 @@
     long_description = f.read()
 
 setup(
-    name="seidemann-lab-to-nwb",
+    name="seidemann_lab_to_nwb",
```

Comment:
For this I was following nwb-conversion-tools standard, what's the reasoning behind this?